UnityProject/Assets/Scripts/Systems/Chemistry/ReagentMix.cs

```diff
@@ -480,7 +483,7 @@ public void Multiply(float multiplier)
 				foreach (var key in reagents.m_dict.Keys)
 				{
 					var nuber = reagents.m_dict[key];
-					nuber = nuber * multiplier;
+					nuber = (float)Math.Round(nuber * multiplier, 2);
```

Comment:
The only issue I see is with stuff like health where, blood has like a really small amount x and y

Reply:
this might result in chemical duplication

Reply:
/destruction

Reply:
Does health use reagent amounts smaller than 0.01u? didnt think it did, but even so I could just change it to 3dps.

Reply:
it might do since, each body part consumes a small amount of nutriment and oxygen and stuff,  so that might not play nicely

Reply:
Would 3dps be enough? I highyl doubt we dealing with numbers smaller than 0.001u

Reply:
I feel like it would upset the maths for the automatic balance of the organs, since it does the maths to work out how much each organ should consume for nutriment and stuff,  maybe have it so beakers and transferring rounds it if you're using reagent containers, but not reagent mix directly

Reply:
The issue is, we have stuff happening right now like:
20u - 10u = 9.99998u, which makes precise chem amounts impossible to work with. Also does any organ use less than 0.001u per second? That seems obsurdly low

Reply:
Hmmm, If I make it 4dp, might still prevent some floating point erros and nutrient consumption will be still 0.0004. which is only 5% lower than current consumption. I dont think a 5% decrease in consumption will change much at all, an in return chemistry will be easier to work with

Reply:
ok, fair enough but I feel like it might cause issues with the small consumption, I would rather go for whenever you do reagent transfers as a player it Rounds it, up to you though

Reply:
Its not just reagent transfers though, its whenever reagents are added or removed, floating point precision makes you get stuff like 1u + 2u = 3.00001u.
Also with small consumption, 4dps is incredibly small, if something is consuming less than 0.0001u in a given timeframe, just extend the timeframe by that point

Reply:
yeah, Fair enough